viz-lib/src/visualizations/chart/plotly/updateAxesInversion.js

```diff
@@ -0,0 +1,32 @@
+import { each, isObject } from "lodash";
+export default function updateAxesInversion(seriesList, layout, options) {
```

Comment:
So, @rafawendel I know I told you to create a new `updatesAxesInvertion`, but there is one thing I noticed in here 😬. What happens is that this is conflicting with `updatesYRanges` and the operation we do is a Lodash `merge`. Since we are "renaming" the variable, we end up merging in the wrong attributes. One failure example happens when you manually set ranges:

<img width="1728" alt="Screen Shot 2020-09-10 at 20 17 14" src="https://user-images.githubusercontent.com/3356951/92822519-a19e9e80-f3a2-11ea-84cf-f35db43de8ac.png">


The simplest way I see to fix it is to move this logic to `updateYRanges` (will ofc have to rename it to something like `updateAxes` or whatever you think makes sense):

```js
export default function updateAxes(plotlyElement, layout, options, seriesList) {
  const updates = {};
  // [...]

  return [
    updates,
    () => {
      // Update Y Ranges
      // [...]

      // Inverted Axes
      if (options.invertedAxes) {
        each(seriesList, series => {
          series.orientation = "h";
          const { x, y } = series;
          series.x = y;
          series.y = x;
        });

        const { xaxis } = layout;
        const { yaxis, yaxis2 } = updates;

        if (isObject(xaxis) && isObject(yaxis)) {
          updates.xaxis = yaxis;
          updates.yaxis = xaxis;
        }
        if (isObject(yaxis2)) {
          updates.yaxis2 = null;
        }
      }

      return [updates, null]; // no further updates
    },
  ];
}
```

We can also use a third level of `Plotly.relayout`, but I guess having the code in the same file is better than that 😅